### DISCUSSION TITLE
东南大学（九龙湖校区）

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#35] [University]
> 回答来自群友

**宿舍是上床下桌吗**
是上床下桌

**教室和宿舍有没有空调**
教室和宿舍有空调

**有独立卫浴吗?没有独立浴室的话，澡堂离宿舍多远**
宿舍有独立卫浴

**有早自习、晚自习吗**
大一有晚自习

**有晨跑吗**
每学期45次晨跑

**每学期跑步打卡的要求是多少公里，可以骑车吗**
大概1-2公里，可以走路，路线上都有老师守着，不能骑车

**寒暑假放多久，每年小学期有多长**
正常放假。。。暑假小学期3-4周，记不清了

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
学校允许点外卖，送到宿舍大门口

**学校附近有地铁站吗**
宿舍到地铁站大概3公里。

**宿舍楼有洗衣机吗**
每栋楼一楼有公共洗衣机，可以悄悄买洗衣机，但是会查宿舍需要藏起来，查到不好办。

**校园网怎么样**
还行，将个烂就

**每天断电断网吗，几点开始断**
不断网断电，十一点熄灯

**食堂价格贵吗，会吃出异物吗**
难吃，贵，学校偏僻，外卖也不多。校内没有商业街，只能吃食堂。

**洗澡热水供应时间**
随时有

**校园内可以骑电瓶车吗，电池在哪能充电待补充**
可以骑车，没地方充电，可以悄悄找地方冲，我也不太懂。

**宿舍限电情况**
4000w，实际不止

**通宵自习有去处吗**
有专门开设通宵自习的教室，需要占座

**大一能带电脑吗**
不能带，辅导员不定期查宿舍

**学校里面用什么卡，食堂怎么消费**
校园卡，刷卡

**学校发银行卡吗**
发

**学校超市怎么样**
只有3个超时，东西种类不多，水泄不通，排队是常有的事

**学校的收发快递政策怎么样**
有菜鸟驿站

**共享单车**
比较多

**门禁**
晚上会锁门，需要给阿姨叫起来开门

**晚上查寝吗，可以晚归吗**
可以，只要把阿姨叫起来，晚上不查寝

总结，学校周边都是工业园区，最近的商场要打车5公里，总之如果想要放松，是一件成本蛮高的事情。

**Comment by Apple1203** [comment]
这是谁回复的 都多少年了 现在跑操早就变成集体跑操了 人脸识别不能走路 宿舍不限电 超市种类还不错 外卖送到学校门口自己去拿 暑假寒假也就各一个月 离商场打车也就十几二十块钱 也不远

**Reply by Apple1203** [reply]
大一还不能带电脑？只有军训期间不让掏出来放在桌面上 不然学计算机的学啥了

**Reply by Apple1203** [reply]
分宿舍区门禁 有的宿舍区不锁门

**Comment by BlowerLacer** [comment]
九龙湖三个食堂很多花样，真不至于不好吃，贵确实有点贵，但是便宜的窗口也有
大一不能带电脑纯鬼扯的，总不能英语pre用空手搓ppt吧

**Comment by ColoZhao** [comment]
九龙湖研究生寝室，兰台那里，似乎没有独立卫浴？
